tests/role_moderation_test.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+u"""test moderated sim types
+
+:copyright: Copyright (c) 2019 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+from pykern.pkcollections import PKDict
+import os
+import pytest
+
+def setup_module(module):
+    os.environ.update(
+        SIREPO_FEATURE_CONFIG_MODERATED_SIM_TYPES='elegant',
+        SIREPO_AUTH_ROLE_MODERATION_MODERATOR_EMAIL='vagrant@localhost.localdomain',
```

Comment:
Use `os.getlogin()` instead of `vagrant`.

Reply:
8ef331f5ed8d6d578701a4374d899b1b06f51658